app.py

```diff
@@ -1,12 +1,132 @@
 """Blogly application."""
 
-from flask import Flask
-from models import db, connect_db
+from flask import Flask, request, redirect, flash
+from flask.templating import render_template
+from models import db, connect_db, User, Post
 
 app = Flask(__name__)
-app.config['SQLALCHEMY_DATABASE_URI'] = 'postgresql:///blogly'
+app.config['SQLALCHEMY_DATABASE_URI'] = 'postgresql://localhost/blogly?user=postgres&password=postgresql'
 app.config['SQLALCHEMY_TRACK_MODIFICATIONS'] = False
 app.config['SQLALCHEMY_ECHO'] = True
+app.config['SECRET_KEY'] = 'flaskblogly'
 
 connect_db(app)
-db.create_all()
+
+
+@app.route('/')
+def home_page():
+    """"list all users"""
+    users = User.query.order_by(User.first_name, User.last_name).all()
+    return render_template('user.html', users=users)
+
+
+@app.route('/<int:user_id>')
```

Comment:
I would precede all user-related routes with `/user/` - like you have it down below for adding a post for a user.